docs/proposal-group-trigger.md

```diff
@@ -116,15 +116,18 @@ Off-group prerequisites and outputs are marked with target icons.
 Cylc 7 intervention:
 
 ```bash
-cylc reset FAMILY.1 -s waiting
+cylc reset FAMILY.1 -s waiting <workflow>
 ```
 
 Cylc 8.4.0 intervention:
 
+(Aside: as noted above, we assume here that inactive task globbing works;
+to actually run with 8.4 replace `FAMILY` with explicit member names.)
```

Comment:
I'm not sure what we gain by documenting a historical intervention that never worked?